testutils/file.go

```diff
@@ -30,12 +31,79 @@ func TempDir(dir, pattern string) (string, error) {
 	return dir, err
 }
 
-// TempFile returns a new unique temporary file using the given name.
-func TempFile(tb testing.TB, name string) *os.File {
+// TempFile creates a unique temporary file named "something.txt" or fails the test if it
+// cannot. It returns the file and a clean-up function.
+func TempFile(tb testing.TB) (*os.File, func()) {
 	tb.Helper()
-	tempFile := filepath.Join(tb.TempDir(), name)
+	tempFile := filepath.Join(tb.TempDir(), "something.txt")
```

Comment:
I can't find any usages of this helper where we need to specify a filename, so I'm just removing that input for convenience-sake. 

It's easy enough to add another helper that allows specific file names in the future if we ever need it.